tests/unit/30_core_1.bats

```diff
@@ -128,6 +128,16 @@ build_nginx_config() {
   assert_success
 }
 
+@test "(core) run command from Procfile" {
+  deploy_app
+  run /bin/bash -c "dokku run $TEST_APP console -c 'echo hi' | tail -n 1"
+  echo "output: "$output
+  echo "status: "$status
+
+  assert_success
+  assert_output "hi"
```

Comment:
Yes, it seems to be an issue specifically with quoted named arguments. I've updated the test case to not rely on that and I will open a separate issue on herokuish.